# request
string action
string collection_name
string data
---
# response
bool status
string result